velox/functions/prestosql/aggregates/tests/MergeAggregateTest.cpp

```diff
@@ -83,11 +88,96 @@ class MergeAggregateTest : public AggregationTestBase {
         actual.min() == expected.min() && actual.max() == expected.max() &&
         actual.compression() == expected.compression();
   }
+
+  template <typename T>
+  facebook::velox::functions::qdigest::QuantileDigest<T, std::allocator<T>>
+  createQDigest(const char* inputData) {
+    // Use standard allocator with uninitialized maxError
+    facebook::velox::functions::qdigest::QuantileDigest<T, std::allocator<T>>
+        qdigest(
+            std::allocator<T>(),
+            facebook::velox::functions::qdigest::kUninitializedMaxError);
+    qdigest.mergeSerialized(inputData);
```

Comment:
You can use the `QuantileDigest(const Allocator& allocator, const char* serialized)` constructor.